src/js/components/TaskListComponent.jsx

```diff
@@ -167,10 +170,7 @@ var TaskListComponent = React.createClass({
                 </th>
             </tr>
           </thead>
-          <PagedContentComponent
```

Comment:
```PagedContentComponent``` took the _whole_ list of of ```TaskListItems```-components generated by ```getTasks```, even if you display only 8 of them at once, what is not scaleable.
Now we are prefiltering the tasks in the lazy-sequence-chain directly and only the TaskListItems will be generated that will be shown.